ShopLens/ShopLensForms/ShopLens.cs

```diff
@@ -57,21 +57,26 @@ private void CommandRecognized_WhatIsThis(object sender, EventArgs e)
         {
             if (live_video.Image != null)
             {
-                Bitmap image = (Bitmap)live_video.Image.Clone();
-                capture_picture.Image = (Bitmap)live_video.Image.Clone();
+                var image = (Image)live_video.Image.Clone();
+                capture_picture.Image = image;
 
                 var ms = new MemoryStream();
                 image.Save(ms, System.Drawing.Imaging.ImageFormat.Jpeg);
 
                 var classificationResults = _imageClassifying.ClassifyImage(ms.ToArray());
 
                 var resultStrings = classificationResults.Select(pair => $"{pair.Key} - {(int)(pair.Value * 100)} percent.");
```

Comment:
I think `resultStrings` is not needed anymore

Reply:
Yeah, it's not.